doc/tutorials/basic/templates.md

```diff
@@ -0,0 +1,226 @@
+# Use Templates
+
+In this tutorial, we will use *pre-made templates* to easily layout an app with a *header*, *sidebar*, and *main* area:
+
+- Templates are available in the `pn.template` namespace
+- Templates can be found in the [Templates Section](https://panel.holoviz.org/reference/index.html#templates) of the [Component Gallery](../../reference/index.md).
+- Templates are highly customizable
+
+## Install the Dependencies
+
+Please make sure [Vega-Altair](https://altair-viz.github.io/) is installed.
+
+::::{tab-set}
+
+:::{tab-item} conda
+:sync: conda
+
+``` bash
+conda install -y -c conda-forge altair
+```
+
+:::
+
+:::{tab-item} pip
+:sync: pip
+
+``` bash
+pip install altair
+```
+
+:::
+
+::::
+
+## Create a Hello World App
+
+Copy the code below to a file `app.py`.
+
+```python
+import panel as pn
+
+pn.extension()
+
+pn.template.FastListTemplate(
+    title="Hello World",
+    sidebar=["# Hello Sidebar", "This is text for the *sidebar*"],
+    main=["# Hello Main", "This is text for the *main* area"],
+).servable()
+```
+
+Serve the app with
+
+```bash
+panel serve app.py --autoreload
+```
+
+It should look like below
+
+![Hello World FastListTemplate App](../../_static/images/templates_hello_world.png)
+
+:::{note}
+The code refers to
+
+- `pn.template.FastListTemplate`: The specific *template* to use.
+- `title`: An optional title to be displayed in the top *header*.
+- `sidebar`: An optional list of objects to display in the left *sidebar*.
+- `main`: A list of objects to display in the *main* area.
+
+See the [FastListTemplate](../../reference/templates/FastListTemplate.ipynb) *reference guide* for more configuration options.
+:::
+
+:::{tip}
+Panel ships with a large collection of built-in templates. There is even a [`Slides`](../../reference/templates/Slides.ipynb) template.
+:::
+
+Spend a couple of minutes checking out the [Templates Section](https://panel.holoviz.org/reference/index.html#templates) of the [Component Gallery](../../reference/index.md). Then return here.
+
+[![Templates Section](../../_static/images/templates_section.png)](https://panel.holoviz.org/reference/index.html#templates)
+
+## Use a Template in a Notebook
+
+:::{note}
```

Comment:
I have changed to a Warning.

I have tried to explain the *why* below.

Currently *templates* do not display in a notebook we hope to support this one day. Please upvote [Issue #2677](https://github.com/holoviz/panel/issues/2677) if this use case is important to you.

Feel free to propose a better formulation.

Reply:
Weird, is it normal for a Warning admonition to display as blue?

<img width="1040" alt="image" src="https://github.com/holoviz/panel/assets/35924738/7517828c-7cf4-4726-94ca-41c94b595814">

Can you put it at the end of the `Use a Template in a Notebook` section? The Panel Preview allows people to use templates "in a notebook" sort of.

Reply:
I've changed from `:::{admonition} Warning` to `:::{warning}`, I believe this will fix the blue color.

I've moved the warning to the end of the section.